src/libs/common/include/storm_d3dx9.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#ifndef _WIN32
+
+#include <d3d9.h>
+#include <algorithm>
```

Comment:
why `<algorithm>` needed here?

Reply:
Moved to storm_d3dx9.cpp, where it actually used.